apps/web/src/components/EmptyMessage/EmptyMessage.tsx

```diff
@@ -0,0 +1,22 @@
+import { Flex, Text } from "@chakra-ui/react";
+
+type EmptyMessageProps = {
+  title: string;
+  subtitle: string;
+};
+
+export const EmptyMessage = ({ title, subtitle }: EmptyMessageProps) => (
+  <Flex alignItems="center" justifyContent="center" flexDirection="column" gap="12px" width="full">
+    <Text size={{ base: "xl", lg: "2xl" }} variant="bold">
+      No ‘{title}’ to show
```

Comment:
ticks seem strange 🤔 are those `'`?

Reply:
It is directly copied from Figma, but I think those quotes can be easily removed because the text looks good even without them
<img width="329" alt="image" src="https://github.com/trilitech/umami-v2/assets/169451151/bbd5eea9-6773-401a-a5c1-af9b83fc551b">

Reply:
Ah, fine then, let's leave it as it is